test/integration/index.js

```diff
@@ -35,6 +36,7 @@ const cleanup = async () => {
         },
         { retries: 3 }
       )
+      // await deleteStaleEnvironments()
```

Comment:
I'd place this command before creating an environment for the current test.

Reply:
And wrap it in `try catch`, we don't want to fail the whole integration script if any deletion fails.